op-devstack/sysgo/config.go

```diff
@@ -0,0 +1,12 @@
+package sysgo
+
+import "strconv"
+
+const SysgoMetricsEnabledEnvVar = "SYSGO_METRICS_ENABLED"
+
+func AreMetricsEnabled() bool {
```

Comment:
Only used inside this package, no need to export.
```suggestion
const sysgoMetricsEnabledEnvVar = "SYSGO_METRICS_ENABLED"

func areMetricsEnabled() bool {
```
(similar for the other constants, etc.)

Reply:
https://github.com/ethereum-optimism/optimism/pull/18017/commits/bd62f252391c74523af1228d5f7ec83e7383d9cb

Please let me know if there's anything I missed

Reply:
`GetAvailableLocalPort` I think.